.github/workflows/update-charts.yml

```diff
@@ -0,0 +1,125 @@
+          NEWVERSION=${{github.event.client_payload.version}}
+
+          VALID=$(semver validate $OLDVERSION)
+          if [[ $VALID == "invalid" ]]; then
+                  exit 1
+          fi
+
+          VALID=$(semver validate $NEWVERSION)
+          if [[ $VALID == "invalid" ]]; then
+                  exit 1
+          fi
+
+          UPDATE_TYPE=$(semver diff $OLDVERSION $NEWVERSION)
+          echo "update_type=$UPDATE_TYPE" >> $GITHUB_OUTPUT
+          echo "update_type=$UPDATE_TYPE"
+          cat $GITHUB_OUTPUT
+
+  patch-update:
+    name: Patch release updates
+    runs-on: ubuntu-latest
+    needs:
+      - check-update-type
+    if: needs.check-update-type.outputs.update_type == 'patch'
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Set environment variables
+        uses: actions/github-script@v6
+        with:
+          script: |
+            core.exportVariable("UPDATECLI_GITHUB_OWNER", context.repo["owner"])
+            core.exportVariable("UPDATECLI_CHART_VERSION", context.payload.client_payload.version)
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Download CRDS
+        if: endsWith(github.event.client_payload.repository, 'kubewarden-controller')
+        uses: actions/github-script@v6
+        with:
+          script: |
+            let repository = context.payload.client_payload.repository
+            if (repository.endsWith("kubewarden-controller")) {
+              let crds_asset_id = context.payload.client_payload.crds_asset_id
+              console.log(`Fetching asset ID: ${crds_asset_id}`)
+              let repository_split = context.payload.client_payload.repository.split("/")
+              let owner = repository_split[0]
+              let repository = repository_split[1]
+              let asset = await github.rest.repos.getReleaseAsset({
+                      owner: owner, repo: repository, asset_id: crds_asset_id, headers:{
+                              accept: "application/octet-stream"},
+              })
+              let fs = require('fs');
+              fs.writeFileSync("crds.tar.gz", Buffer.from(asset.data))
+            }
+
+      - name: Update CRDS
+        if: endsWith(github.event.client_payload.repository, 'kubewarden-controller')
+        id: update_crds
+        run: |
+          tar -xvf crds.tar.gz
+          find . -maxdepth 1 -name "*policyserver*" -exec  mv \{\} charts/kubewarden-crds/templates/policyserver.yaml \;
+          find . -maxdepth 1 -name "*admissionpolicies*" -exec  mv \{\} charts/kubewarden-crds/templates/admissionpolicy.yaml \;
+          find . -maxdepth 1 -name "*clusteradmissionpolicies*" -exec  mv \{\} charts/kubewarden-crds/templates/clusteradmissionpolicies.yaml \;
+
+          set +e
+          git diff --exit-code --no-patch charts/kubewarden-crds
+          echo "must_update_crds_chart=$?" >> $GITHUB_OUTPUT
+          git diff
```

Comment:
@olblak how can I add these files in the branch used to open the PR? I can open a PR with the files changed using updatecli (yaml files). However, I could find a workaround to add files from external source in the commit

Reply:
I need to try it but I would move 
```
          tar -xvf crds.tar.gz
          find . -maxdepth 1 -name "*policyserver*" -exec  mv \{\} charts/kubewarden-crds/templates/policyserver.yaml \;
          find . -maxdepth 1 -name "*admissionpolicies*" -exec  mv \{\} charts/kubewarden-crds/templates/admissionpolicy.yaml \;
          find . -maxdepth 1 -name "*clusteradmissionpolicies*" -exec  mv \{\} charts/kubewarden-crds/templates/clusteradmissionpolicies.yaml \;
```

to a shell script then call that shell script from the target of kind "shell"